### PR TITLE
Add a post about ChatGPT in Galaxy Interactive tool

### PR DESCRIPTION
**Pull request by anuprulez**

### CODE REVIEW

**Comment by anuprulez** [comment]
ping @bgruening thanks!

File: content/news/2023-08-14-chatGPT/index.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+
+
+### Pricing for OpenAI token
+
+The cost of using OpenAI models depends on factors such as the type of model (LLM) used and the number of tokens. The [price chart](https://openai.com/pricing) describes the detailed cost of using different LLMs. Models such as GPT4 are more expensive than GPT-3.5 Turbo. Usage cost also depends on the number of tokens/words used in the query and its response. Refer to the [tokens guide](https://platform.openai.com/docs/guides/gpt/managing-tokens) for more understanding. A straightforward example of how tokens are managed inside models is - the string "ChatGPT is great!" is encoded into six tokens: ["Chat", "G", "PT", " is", " great", "!"]. Using the GPT-3.5 Turbo model for 1000 tokens costs $0.0015.
+
+
+### How to access on Galaxy Europe:
+
+1. Acquire an OpenAI access token by creating an [OpenAI](https://openai.com/) account. It should be noted that using OpenAI models requires a premium/paid account.
```
Comment (bgruening) [inline content/news/2023-08-14-chatGPT/index.md:41]:
```suggestion
1. Acquire an OpenAI access token by creating an [OpenAI](https://openai.com/) account. It should be noted that using OpenAI models requires a premium/paid account from OpenAI.
```

File: content/news/2023-08-14-chatGPT/index.md [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+
+
+![Jupyternaut 2](./chatgpt_2.png)
+
+Another way to use ChatGPT is via a notebook using "magic commands `%%ai`". Refer to the following image.
+
+![ChatGPT in a notebook](./chatgpt_4.png)
+
+
+### Pricing for OpenAI token
```
Comment (bgruening) [inline content/news/2023-08-14-chatGPT/index.md:34]: This is probably soon outdated, so we really want to include this here?

File: content/news/2023-08-14-chatGPT/index.md [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+
+The cost of using OpenAI models depends on factors such as the type of model (LLM) used and the number of tokens. The [price chart](https://openai.com/pricing) describes the detailed cost of using different LLMs. Models such as GPT4 are more expensive than GPT-3.5 Turbo. Usage cost also depends on the number of tokens/words used in the query and its response. Refer to the [tokens guide](https://platform.openai.com/docs/guides/gpt/managing-tokens) for more understanding. A straightforward example of how tokens are managed inside models is - the string "ChatGPT is great!" is encoded into six tokens: ["Chat", "G", "PT", " is", " great", "!"]. Using the GPT-3.5 Turbo model for 1000 tokens costs $0.0015.
+
+
+### How to access on Galaxy Europe:
+
+1. Acquire an OpenAI access token by creating an [OpenAI](https://openai.com/) account. It should be noted that using OpenAI models requires a premium/paid account.
+2. Apply for using GPU JupyterLab using http://usegalaxy.eu/gpu-request. Ignore this step if you have access to GPU JupyterLab in Galaxy Europe.
```
Comment (bgruening) [inline content/news/2023-08-14-chatGPT/index.md:42]:
```suggestion
2. If you do not have already, please apply for using GPU JupyterLab via http://usegalaxy.eu/gpu-request.
```

Comment (bgruening) [reply content/news/2023-08-14-chatGPT/index.md:42]: Should that maybe the first step?

File: content/news/2023-08-14-chatGPT/index.md [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+3. Go to GPU JupyterLab tool in Galaxy Europe and choose tool version 0.2. Currently, it has been deployed on this version only. It may be deployed on other interactive tools (ITs) in Galaxy, serving Jupyter notebooks online.
```
Comment (bgruening) [inline content/news/2023-08-14-chatGPT/index.md:43]: link to the GPU Jupterlab?

Comment (bgruening) [reply content/news/2023-08-14-chatGPT/index.md:43]:
```suggestion
3. Go to GPU JupyterLab tool in Galaxy Europe and choose tool version 0.2. 
```

Comment (bgruening) [reply content/news/2023-08-14-chatGPT/index.md:43]: But I would add a direct link

File: content/news/2023-08-14-chatGPT/index.md [hunk 5]
```diff
@@ -0,0 +1,55 @@
...
+4. Open the JupyterLab and click the "chat" icon at the bottom of the left panel. Choose the model you want to use and provide an OpenAI access token. It gives a chatbox to write questions. Refer to images 1 and 2.
+5. Creating a regular notebook is Another way to use these models. Type the following commands in the same order in a notebook:
```
Comment (bgruening) [inline content/news/2023-08-14-chatGPT/index.md:45]:
```suggestion
5. Creating a regular notebook is another way to use these models. Type the following commands in the same order in a notebook:
```

File: content/news/2023-08-14-chatGPT/index.md [hunk 6]
```diff
@@ -0,0 +1,55 @@
...
+
+    a. `%load_ext jupyter_ai`
+ 
+    b. `%env OPENAI_API_KEY=<<use your OpenAI Key>>`
+ 
+    c. `%%ai chatgpt`
+
+    d. `Write a program for plotting heatmap using matplotlib`
+
+6. One example notebook is provided at `/notebooks/test-Generative-AI.ipynb`. Click [Jupyter AI](https://jupyter-ai.readthedocs.io/en/latest/users/index.html) for further examples and details.
```
Comment (bgruening) [inline content/news/2023-08-14-chatGPT/index.md:55]:
```suggestion
6. An example is provided at `/notebooks/test-Generative-AI.ipynb` inside the notebook. 

For further examples and details see the [Jupyter AI page](https://jupyter-ai.readthedocs.io/en/latest/users/index.html).
```

File: content/news/2023-08-14-chatGPT/index.md [hunk 7]
```diff
@@ -0,0 +1,55 @@
...
+
+![ChatGPT in a notebook](./chatgpt_4.png)
+
+
+### Pricing for OpenAI token
+
+The cost of using OpenAI models depends on factors such as the type of model (LLM) used and the number of tokens. The [price chart](https://openai.com/pricing) describes the detailed cost of using different LLMs. Models such as GPT4 are more expensive than GPT-3.5 Turbo. Usage cost also depends on the number of tokens/words used in the query and its response. Refer to the [tokens guide](https://platform.openai.com/docs/guides/gpt/managing-tokens) for more understanding. A straightforward example of how tokens are managed inside models is - the string "ChatGPT is great!" is encoded into six tokens: ["Chat", "G", "PT", " is", " great", "!"]. Using the GPT-3.5 Turbo model for 1000 tokens costs $0.0015.
+
+
+### How to access on Galaxy Europe:
```
Comment (bgruening) [inline content/news/2023-08-14-chatGPT/index.md:39]: Isn't that section a bit redundant with the general usage section?

Comment (anuprulez) [reply content/news/2023-08-14-chatGPT/index.md]: @bgruening, thanks a lot for your comments. I have update the blog according to your comments

**Review by bgruening — APPROVED** [review]

**Comment by bgruening** [comment]
Cool!

**Pull request merged**